docs/querying/api.md

```diff
@@ -575,6 +575,7 @@ GET /api/v1/rules
 
 URL query parameters:
 - `type=alert|record`: return only the alerting rules (e.g. `type=alert`) or the recording rules (e.g. `type=record`). When the parameter is absent or empty, no filtering is done.
+- `match[]=<label_selector>`: Repeated label selector argument that select rules which have labels that satisfy the selectors. Optional.
```

Comment:
```suggestion
- `match[]=<label_selector>`: Repeated label selector argument that selects rules which have configured labels that satisfy the selectors. Note that alerting rules it matches the top-level potentially templated labels, not the rendered labels of active alerts (instances). Optional.
```

Reply:
Updated docs with a slight change, we only match non-templated labels.